src/lib.rs

```diff
@@ -63,9 +63,34 @@ pub use cqe::{CompletionQueue, CompletionQueueEvent};
 pub use registrar::Registrar;
 
 bitflags::bitflags! {
+    /// `IoUring` initialization flags for advanced use cases.
+    ///
+    /// ```
```

Comment:
I'm getting permission denied when I run this test as my normal user (running Linux 5.2.11).

Maybe this test should be `no_run` if some of these configurations require special privileges?

Reply:
Right, thank you. Should be fixed by 43271ac